### PR TITLE
FKM-14. Document display.

### PR DESCRIPTION
**Pull request by dporplenko**

### CODE REVIEW

**Comment by dporplenko** [comment]
test this please

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/65/

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install [hunk 1]
```diff
@@ -791,3 +791,13 @@ function favrskov_helper_update_7039() {
...
+
+/**
+ * Revert features for wysiwyg dispaly.
+ */
+function favrskov_helper_update_7041() {
+    array_map('features_revert_module', array(
+        'config_media',
+        'fields_shared',
+    ));
+}
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install]: Add new line at the end of file.

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/66/

**Pull request merged**